### PR TITLE
Adds ci script

### PR DESCRIPTION
**Pull request by SHession**
## What does this change?

This PR changes the CI/CD process to use a script instead of Team City steps.

This matches our best practice for our applications and will allow us to experiment with changes to our CI without affecting main. 

## How to test

Does the project build and deploy successfully? (We have updated TC to use the script and deployed to CODE)

## Images

image

### CODE REVIEW

File: scripts/ci.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    nvm install
+    nvm use
+}
+
+globalJsDependencies() {
+    npm install -g yarn
+}
+
+yarnSetup() {
+    yarn install --force
```
Comment (akash1810) [inline scripts/ci.sh:18]: This doesn't yield a deterministic build as the lockfile get's completely ignored - `yarn install --frozen-lockfile` is preferred. From https://classic.yarnpkg.com/en/docs/cli/install/:

> If you need reproducible dependencies, which is usually the case with the continuous integration systems, you should pass `--frozen-lockfile` flag.

**Review by akash1810 — COMMENTED** [review]
Could I tempt you towards [GitHub Actions](https://docs.google.com/document/d/1Xz-gzptICIza0oio0Kza3LWg2a6n_015o0MPu0LGfuU/edit)? More than happy to pair!

File: scripts/ci.sh (hunk 1 above)
Comment (SHession) [reply scripts/ci.sh]: Good advice, thanks @akash1810!

**Comment by SHession** [comment]
This PR is no longer needed after the switch to GitHub actions 🎉

**Pull request closed without merging**